src/eth/storage/csv/csv_exporter.rs

```diff
@@ -516,11 +518,22 @@ fn csv_writer(base_path: &'static str, number: BlockNumber, headers: &[&'static
         .from_path(path)
         .context("failed to create csv writer")?;
 
-    writer.write_record(headers).context("fai;ed to write csv header")?;
+    writer.write_record(headers).context("failed to write csv header")?;
 
     Ok(writer)
 }
 
+/// Convert a byte sequence to a `bytea` representation that is parseable by Postgres.
+fn to_bytea(bytes: impl AsRef<[u8]>) -> String {
+    let hex = hex::encode(bytes.as_ref());
+    format!("\\x{hex}")
+}
+
+fn u256_to_bytea(integer: U256) -> String {
+    let bytes: Vec<u8> = integer.as_ref().iter().copied().rev().flat_map(u64::to_be_bytes).collect();
```

Comment:
I need more eyes on this `u64::to_be_bytes`.

Reply:
We tested that it produces the expected result in my local machine, but could it change for other machines? (does it even matter?)

Reply:
we already use `to_be_bytes` in other parts of the code ([example](https://github.com/cloudwalk/stratus/blob/b8c81b22a4a7ad6b224e35ba67fd48d2cddba931/src/eth/primitives/block_number.rs#L166)). to this day it never was a problem.